pkg/controller/signup_test.go

```diff
@@ -507,7 +470,7 @@ func (s *TestSignupSuite) TestVerifyPhoneCodeHandler() {
 
 		require.Equal(s.T(), "Not Found", bodyParams["status"])
 		require.InDelta(s.T(), float64(404), bodyParams["code"], 0.01)
-		require.Equal(s.T(), "usersignups.toolchain.dev.openshift.com \"jsmith\" not found: user not found", bodyParams["message"])
+		require.Equal(s.T(), "usersignups.toolchain.dev.openshift.com \"fdebf2d6-jsmithkubesaw\" not found: user not found", bodyParams["message"])
```

Comment:
where is the "fdebf2d6" part coming from? Maybe it'd be worth explaining it with a comment. I personally haven't yet figured it out.

Reply:
it's the result of encoding the username - see https://github.com/codeready-toolchain/toolchain-common/blob/8ef21eb833fa3176af6cd0cf3842a8b64dee7dfa/pkg/usersignup/usersignup.go#L82-L123
added a comment: 7e48ae0